docs/cookbook/recipe_sonata_admin_without_user_bundle.rst

```diff
@@ -0,0 +1,451 @@
+SonataAdmin without FOSUserBundle/SonataUserBundle
+==================================================
+
+Here we will explain how you can use SonataAdmin with Symfony Guard instead
+of using FOSUserBundle or SonataUserBundle.
+
+Code used in this short guide can be found `here <https://github.com/kunicmarko20/guard-talk-example>`__
+with support for different Symfony versions.
+
+.. note::
+    This article assumes you are using Symfony 4.
+
+The recipe
+----------
+
+Things we will need to do:
+
+- User Entity
+- UserProvider
+- LoginForm
+- LoginAuthenticator
+- LoginController
+- Setup firewall in `security.yaml`
+- Login template resembling Sonata style
```

Comment:
shall we make the TOC clickable?

Reply:
Any examples on how to do this?

Reply:
https://stackoverflow.com/questions/15394347/adding-a-cross-reference-to-a-subheading-or-anchor-in-another-page/15670611#15670611

Reply:
added